Common/tests/Pollster/PollsterTest.cs

```diff
@@ -568,4 +575,63 @@ await pollster.Init(CancellationToken.None)
     Assert.AreSame(string.Empty,
                    pollster.TaskProcessing);
   }
+
+
+  [Test]
+  public async Task UnavailableWorkerShouldFail()
```

Comment:
This should most likely be merged with the previous test.

Reply:
The only difference seems to be:
```c#
    Assert.AreEqual(TaskStatus.Submitted,
                    (await testServiceProvider.TaskTable.GetTaskStatus(new[]
                                                                       {
                                                                         tuple.taskSubmitted,
                                                                       },
                                                                       CancellationToken.None)
                                              .ConfigureAwait(false)).Single()
                                                                     .Status);
```

I believe this test should also be valid on all the other cases.